pkg/util/sql/schema.sql

```diff
@@ -1,10 +1,20 @@
 -- Users
 CREATE TABLE IF NOT EXISTS users (
   id BIGSERIAL PRIMARY KEY,
-  identity_token text NOT NULL,
+  login_name varchar(100) NOT NULL,
+  password_hash text NOT NULL,
```

Comment:
is there any better type instead of text for this?

Reply:
Text may be more convenient backwards compatibility, compared to fixed length type like varchar(n). I think text is ok.